src/modules/commander/failsafe/failsafe.cpp

```diff
@@ -501,22 +501,27 @@ void Failsafe::checkStateAndMode(const hrt_abstime &time_us, const State &state,
 	}
 
 	// Battery low failsafe
+	// If battery was low and arming was allowed through COM_ARM_BAT_MIN, don't failsafe immediately for the current low battery warning state
+	const bool warning_worse_than_at_arming = (status_flags.battery_warning > _battery_warning_at_arming);
+	const int32_t low_battery_action = warning_worse_than_at_arming ?
+					   _param_com_low_bat_act.get() : (int32_t)LowBatteryAction::Warning;
```

Comment:
You can get a warning now also if you set COM_LOW_BAT_ACT to 0 - not a big deal in my eyes.

Reply:
`COM_LOW_BAT_ACT` = 0 is emit a warning
https://github.com/PX4/PX4-Autopilot/blob/936eb89edbb07fbb921434d7f58d46305c81f4ca/src/modules/commander/commander_params.c#L271

Reply:
Ups, disregard then.